core/js/src/main/scala/zio/internal/PlatformSpecific.scala

```diff
@@ -85,7 +85,8 @@ private[zio] trait PlatformSpecific {
 
   final def newWeakSet[A]()(implicit unsafe: zio.Unsafe): JSet[A] = new HashSet[A]()
 
-  final def newConcurrentSet[A]()(implicit unsafe: zio.Unsafe): JSet[A] = new HashSet[A]()
+  final def newConcurrentSet[A]()(implicit unsafe: zio.Unsafe): JSet[A]                     = new HashSet[A]()
+  final def newConcurrentSet[A](initialCapacity: Int)(implicit unsafe: zio.Unsafe): JSet[A] = new HashSet[A]()
```

Comment:
did u miss passing the initial capacity to the set's constructor?

Reply:
Ahh yeap, nicely spotted 👍

Reply:
I think there were couple more places like this (native, etc...)